frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx

```diff
@@ -48,6 +48,10 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
 
   fetchGroupPermission = (groupPermissionId) => {
     groupPermissionService.getGroup(groupPermissionId).then((data) => {
+      if (data?.group !== 'admin') {
+        this.setState({ currentTab: 'users' });
+      }
+
       this.setState({
         groupPermission: data,
```

Comment:
```suggestion
      this.setState({
        currentTab: data?.group !== 'admin' ? 'users' : 'apps',
        groupPermission: data,
```

Reply:
Thanks for suggestion! I've updated my code